impeller/aiks/paint.cc

```diff
@@ -60,14 +60,21 @@ std::shared_ptr<Contents> Paint::WithFilters(
     std::shared_ptr<Contents> input) const {
   input = WithColorFilter(input, /*absorb_opacity=*/true);
```

Comment:
Dry by comment on an unrelated item: Instead of booleans for absorb_opacity and is_subpass, I'd rather have a typed enum with AbsorbOpacity::kYes and AbsorbOpacity::kNo and the like.

Reply:
Added an issue about this: https://github.com/flutter/flutter/issues/134681